karma.py

```diff
@@ -1,4 +1,5 @@
 from brutal.core.plugin import BotPlugin, cmd, match
+from brutal.core.utils import change_cmd
```

Comment:
@mrshu which part? Because I was in belief that we were keeping it compatible with `brutal` as well. I mean the `change_cmd` is the first brutal-related component that I haven't seen in `brutal`.

Okay, I'll open an issue for that.